config/sync/core.entity_form_display.paragraph.stanford_page_title_banner.default.yml

```diff
@@ -0,0 +1,39 @@
+uuid: 1c6a81c7-d849-4420-bfcf-f87a8160b5d4
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.stanford_page_title_banner.su_image
+    - field.field.paragraph.stanford_page_title_banner.su_title_banner_header
+    - paragraphs.paragraphs_type.stanford_page_title_banner
+  module:
+    - media_library
+id: paragraph.stanford_page_title_banner.default
+targetEntityType: paragraph
+bundle: stanford_page_title_banner
+mode: default
+content:
+  status:
+    type: boolean_checkbox
+    weight: 2
+    region: content
+    settings:
+      display_label: true
+    third_party_settings: {  }
+  su_image:
+    type: media_library_widget
+    weight: 0
+    region: content
+    settings:
+      media_types: {  }
+    third_party_settings: {  }
+  su_title_banner_header:
```

Comment:
Ya know, I'm thinking about hiding this as well and auto saving the page title into it. One less thing for the user to enter/mess up.

Reply:
no, keep the field just remove it from the form. I will try to put in some logic to fill that field when it is saved.

Reply:
Gotcha, that should be done now.